classes/controller/AdminController.php

```diff
@@ -3650,7 +3650,7 @@ public function getFieldsValue($obj)
      * @param string $key Field name
      * @param int|null $id_lang Language id (optional)
      *
-     * @return string|false
+     * @return false|mixed
```

Comment:
I guess `mixed` is enough here
```suggestion
     * @return mixed
```

Reply:
Oops.